_posts/2024-05-08-CentOS-7-CentOS-Stream-8-deprecation.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: CentOS 7 & CentOS Stream 8 deprecation
+date: 2024-05-08
+github_username: bastelfreak
+twitter_username: bastelsblog
+---
+
+A few operating systems are soon end of life and we won't run acceptance tests
+anymore for those:
+
+* [RedHat 7 - 2024-06-30](https://github.com/voxpupuli/puppet_metadata/blob/dd815b5b350af2936819a7c5a449c1156cfac4b4/lib/puppet_metadata/operatingsystem.rb#L107C17-L107C27)
+* [OracleLinux 7 - 2024-07-01](https://github.com/voxpupuli/puppet_metadata/blob/dd815b5b350af2936819a7c5a449c1156cfac4b4/lib/puppet_metadata/operatingsystem.rb#L53)
```

Comment:
I thought OL "dies" 31.12.2024 ?

Reply:
i.e. you did not drop it in https://github.com/voxpupuli/puppet-jenkins/pull/1106

Reply:
fixed it